src/mix/api/channels.ts

```diff
@@ -30,3 +30,21 @@ export async function renameChannel(client: MixClient, requestParams: ChannelsRe
     data: body,
   })
 }
+
+export async function updateChannel(client: MixClient, requestParams: ChannelsConfigParams): Promise<MixResponse> {
+  debug('updateChannel()')
+  const {projectId, channelId, modes, color,
+    ...searchParams} = requestParams
```

Comment:
There is no searchParams on this endpoint so it can be removed from line 37 and line 47.